library/core/src/primitive_docs.rs

```diff
@@ -1309,10 +1309,12 @@ mod prim_f16 {}
 // FIXME: Is there a better place to put this?
 ///
 /// | `target_arch` | Extra payloads possible on this platform |
-/// |---------------|---------|
-/// | `x86`, `x86_64`, `arm`, `aarch64`, `riscv32`, `riscv64` | None |
+/// |---------------|------------------------------------------|
+// Sorted alphabetically
+/// | `aarch64`, `arm`, `arm64ec`, `loongarch64`, `powerpc` (except when `target_abi = "spe"`), `powerpc64`, `riscv32`, `riscv64`, `s390x`, `x86`, `x86_64` | None |
+/// | `nvptx64` | All payloads |
 /// | `sparc`, `sparc64` | The all-one payload |
```

Comment:
RISC-V always generates an all-one payload.

Reply:
(That seems like a pre-existing issue before this PR then.)

I am confused now, I am pretty sure during the LLVM discussions people said that RISC-V will generate an all-0 payload. And it's not just me, e.g. jcranmer said the same [here](https://discourse.llvm.org/t/stronger-floating-point-nan-guarantees/72165/25).